src/chi/components/tabs/tabs.scss

```diff
@@ -781,6 +781,19 @@
   }
 }
 
+@mixin chevron-icon {
```

Comment:
Would suggest renaming 'chevron-icon' to 'dropdown-icon' as it may not be a chevron in all themes.